### DISCUSSION TITLE
Official ESPHome Component?

### DISCUSSION THREAD

**Discussion by RubenKelevra** [TillFleisch/ESPHome-HLK-LD2450#29] [Q&A] [answered]
Why don't you send this in to make it an official module in ESPHome? 

Thanks for considering!

**Comment by TillFleisch** [comment, marked as answer]
There is already an open PR regarding the LD2450 [see here](https://github.com/esphome/esphome/pull/5674).
As far as my understanding goes, the proposed solution provides more of the sensors functionality (built in detection zones).
Since this repo takes a different approach towards zone detection, there is a reason to keep both solutions around. For the time being, as long as the other PR is still heading towards official integration, I will not open another PR to prevent confusion and this repo will be maintained separately.

**Reply by RubenKelevra** [reply]
Ah thank for the clarification. I thought you might need help with testing or something line that.

The PR seems to have still issues, if your version works fine I mean just give it a go? 🤔
